geoengine/error.py

```diff
@@ -137,6 +137,38 @@ def __str__(self) -> str:
         return f"Spatial reference mismatch {self.__spatial_reference_a} != {self.__spatial_reference_b}"
 
 
+class NoValidUrlException(Exception):
```

Comment:
maybe InvalidUrlException would be better

Reply:
[d825fa0](https://github.com/geo-engine/geoengine-python/pull/102/commits/d825fa04cb26a20bebd9fbab2e1aa3d54d18af97)